apps/studio/src/features/editing-experience/components/PreviewWithoutSitemap.tsx

```diff
@@ -64,6 +64,10 @@ function SuspendablePreview({
         siteMap,
         environment: "production",
         ...siteConfig,
+        search: {
+          type: "localSearch",
+          searchUrl: "/search",
+        },
```

Comment:
some issue with searchsg script in rendering the UI in studio, even with CSP updated. no errors whatsoever.

figured out that we can overwrite it to instead show local search input box for studio users to visualize